modules.d/50plymouth/module-setup.sh

```diff
@@ -33,22 +83,67 @@ depends() {
 # called by dracut
 install() {
     PKGLIBDIR=$(pkglib_dir)
-    if grep -q nash "$dracutsysrootdir""${PKGLIBDIR}"/plymouth-populate-initrd \
-        || [ ! -x "$dracutsysrootdir""${PKGLIBDIR}"/plymouth-populate-initrd ]; then
-        # shellcheck disable=SC1090
-        . "$moddir"/plymouth-populate-initrd.sh
-    else
-        PLYMOUTH_POPULATE_SOURCE_FUNCTIONS="$dracutfunctions" \
-            "$dracutsysrootdir""${PKGLIBDIR}"/plymouth-populate-initrd -t "$initdir"
+    PLUGINDIR=$(plugin_dir)
+
+    inst_multiple readlink
+
+    inst_multiple plymouthd plymouth
+    inst_multiple -o "$PKGLIBDIR"/plymouthd-fd-escrow
+
+    inst_multiple -o "$PLUGINDIR/renderers/drm.so"
+    inst_multiple "$PLUGINDIR/renderers/frame-buffer.so"
+
+    set_theme_dir
+    inst_dir "$PLYMOUTH_THEME_DIR"
+
+    inst_multiple "$PLUGINDIR"/text.so "$PLUGINDIR"/details.so
+    inst_recur "$PLYMOUTH_THEME_DIR"/text
+    inst_recur "$PLYMOUTH_THEME_DIR"/details
+
+    PLYMOUTH_MODULE_NAME=$(grep "ModuleName *= *" "$dracutsysrootdir$PLYMOUTH_THEME_DIR/$PLYMOUTH_THEME_NAME/$PLYMOUTH_THEME_NAME.plymouth" | sed 's/ModuleName *= *//')
+    PLYMOUTH_IMAGE_DIR=$(grep "ImageDir *= *" "$dracutsysrootdir$PLYMOUTH_THEME_DIR/$PLYMOUTH_THEME_NAME/$PLYMOUTH_THEME_NAME.plymouth" | sed 's/ImageDir *= *//')
+
+    inst_multiple -o "$PLUGINDIR/$PLYMOUTH_MODULE_NAME.so"
+    inst_recur "$PLYMOUTH_THEME_DIR/$PLYMOUTH_THEME_NAME"
+
+    if [[ $PLYMOUTH_IMAGE_DIR != "$PLYMOUTH_THEME_DIR/$PLYMOUTH_THEME_NAME" ]] && [[ -n $PLYMOUTH_IMAGE_DIR ]] && [[ -d "$dracutsysrootdir$PLYMOUTH_IMAGE_DIR" ]]; then
+        inst_recur "$PLYMOUTH_IMAGE_DIR"
     fi
 
-    inst_hook emergency 50 "$moddir"/plymouth-emergency.sh
+    inst_multiple \
+        /usr/share/plymouth/plymouthd.defaults \
+        /etc/plymouth/plymouthd.conf
 
-    inst_multiple readlink
+    inst_multiple -o /usr/share/plymouth/themes/default.plymouth
+
+    inst_multiple -o "/usr/share/plymouth/*.png"
 
-    inst_multiple plymouthd plymouth plymouth-set-default-theme
+    inst_multiple -o /etc/system-release
```

Comment:
Why is this needed ? Is this PR based on an earlier version of this module ? 

See also https://github.com/dracut-ng/dracut-ng/commit/d6cef3f28e15a834284786c79f108b7c42481f96

Reply:
It wad not based on an an earlier version of the `plymouth-ng` module, this is straight from #337 using `git rm -r 50plymouth ; git mv 50plymouth-ng 50plymouth`. I did not notice the quoted commit before.